README.md

```diff
@@ -6,6 +6,15 @@ A backend for the Elixir [Logger](https://hexdocs.pm/logger/Logger.html) that:
 
 ## Installation
 
+If not available on Hex:
```

Comment:
Remove `If [available in Hex]` section and keep only instruction for github. We'll add instruction for installation from hex.pm later when we publish it